### ISSUE TITLE
Provide a way for users to suggest sites to scrape

### ISSUE THREAD

**Issue by toby** [toby/playgrub#22]
via JP: Idea: "…currently not supported by Playgrub, click here to request it" -> community ranked/powered scraper request area to playgrub?

**Comment by jphastings** [comment]
Yeah, as people use playgrub more and more there will undoubtedly be thousands of sites that aren't supported. Making the scrapers isn't hard if you have basic Javascript skills so a simple (!) way to see the most requested sites and (communally?) edit scrapers for them would allow people with a few minutes spare to contribute.

My python is mighty rusty, but I'll give some thought to how I'd do this and get back to this thread!
